extensions/vscode/package.json

```diff
@@ -29,7 +29,7 @@
     "Other"
   ],
   "activationEvents": [
-    "workspaceContains:*"
+    "workspaceContains:/"
```

Comment:
Should we change this to just `*` so the extension is always activated whenever VSCode starts up?

Reply:
That sounds like an excellent idea.

Reply:
I tried using the `*` and didn't see any visible improvement in load time, so I think my recommendation is to go with what Mike has currently in this PR.

Reply:
Yes. Both will cause the extension to load when there's a workspace folder open; the difference is that `workspaceContains:/` will not launch the extension in empty workspaces.

Reply:
Fantastic. Thank you both for digging into this.